depobs/worker/main.py

```diff
@@ -21,6 +21,15 @@ def scan_npm_package(package_name: str, package_version: str):
     tasks.scan_npm_package_then_build_report_tree(package_name, package_version)
 
 
+@npm_cli.command("advisories")
+@click.argument("package_name", envvar="PACKAGE_USERNAME")
+def test_package_analysis(package_name: str):
```

Comment:
nit: type signature should be `-> None:` since it doesn't return anything (the other command is missing that too)